h/oauth/jwt_grant.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import json
+
+from oauthlib.oauth2.rfc6749 import errors
+from oauthlib.oauth2.rfc6749.grant_types.base import GrantTypeBase
+
+from h.oauth.jwt_grant_token import JWTGrantToken
+
+
+class JWTAuthorizationGrant(GrantTypeBase):
+    def __init__(self, request_validator, user_svc, domain):
+        self.request_validator = request_validator
+        self.user_svc = user_svc
+        self.domain = domain
+
+    def create_token_response(self, request, token_handler):
```

Comment:
Here as elsewhere, I think it's probably important to document that `request` is an `oauthlib.common.Request`, otherwise the default assumption in this codebase is going to be that it's a `pyramid.request.Request`.